docs/user/commercial/subscriptions.rst

```diff
@@ -0,0 +1,72 @@
+Read the Docs for Business subscription management
+==================================================
+
+We want to make it easy to manage your billing information.
+All :doc:`organization owners </commercial/subscriptions>` can manage the subscription for that organization.
+It's easy in this dashboard to achieve a number of common tasks:
+
+* Update your credit card information.
+* Upgrade, downgrade, or cancel your plan.
+* View, download, and pay invoices.
+* Add additional tax (VAT/EIN) or contact email addresses on your invoices.
```

Comment:
:+1: on this list, we don't spell out all of the features of the subscription dashboard like this, afaik.

Reply:
Yea, useful for SEO as well.